.github/ISSUE_TEMPLATE/01-content-development-general.md

```diff
@@ -15,7 +15,7 @@ A team member will review the proposal and share next steps.
 - Topic description: 
 - Audience (User, Developer, Designer, Contributor, etc.): 
 - Learning objectives (What will the learner be able to do as a result of this content?): 
-- Content type (Tutorial, Online Workshop, Course or Lesson Plan): 
+- Content type (Course, Lesson, Tutorial, Online Workshop, or Lesson Plan): 
```

Comment:
Minor detail, but the previous order of these content types reflected the amount of work needed to create them, and the priority in the Training Team. Psychologically, people are prone to choose the early options if they're not sure what to choose.

Based on that thought, what about reordering these as follows?
```suggestion
- Content type (Online Workshop, Lesson, Course, Tutorial or Lesson Plan): 
```

Reply:
Ah, thanks for clarifying, I was not aware there was a specifically predetermined order.